AUTHORS.rst

```diff
@@ -13,3 +13,7 @@ Community
 Please feel free to add your name on this list if you do a PR!
 
 * José L. Redrejo (jredrejo)
+* `maddevsio <https://github.com/maddevsio>`__ |maddevsio_avatar|
```

Comment:
Links are not part of the policy for LE, glad to include your name and GH nickname, but no links or avatars